jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -178,7 +178,7 @@ def test_wcslink_affine_with_extras(self):
             self.imviz.link_data(align_by='pixels', wcs_fallback_scheme=None)
 
         self.viewer.reset_markers()
-        self.imviz.plugins["Orientation"].delete_subsets()
+        self.imviz.app.delete_subsets()

```

Comment:
should we leave the public-facing API in the test to make sure we have coverage of it (even though it just calls the app method)?